src/components/controls/textbox/Textbox.js

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { noop } from 'lodash';
```

Comment:
Should we add `lodash` as a direct dependency for this package? It's scary that we're depending on a dependency to pull this in for us.